Profiles - ZIB 2017/zib-Pregnancy-PG-extension.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-Pregnancy-PG-extension" />
+  <version value="1.0.0" />
+  <name value="zib-Pregnancy-PG-extension" />
+  <title value="HCIM Pregnancy PG extension" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="An Extension as defined by the Dutch HCIM Pregnancy v3.1(2017EN).&#xD;&#xA;&#xD;&#xA;Pregnancy or gravidity is the condition in which a woman carries a fertilized egg, which grows into an embryo and then a fetus. A pregnancy of between 37 and 42 weeks is considered normal. “Term pregnancy” is 37 to 42 weeks. The term gravidity is also used for the number of times a woman has been pregnant. This and parity - the number of times a woman has given birth - provides insight into the patient’s obstetric history." />
+  <purpose value="# Purpose of HCIM&#xD;&#xA;If the patient is pregnant, it is important to mention this, as it requires specific attention in their care. Pregnancy can influence treatment decisions, the treatment itself and the patient’s care.&#xD;&#xA;&#xD;&#xA;# Purpose of this extension&#xD;&#xA;This extension adds the Partity and Gravidity fields to the zib-Pregnancy profile." />
+  <copyright value="CC0" />
+  <fhirVersion value="3.0.1" />
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <contextType value="resource" />
+  <context value="Condition" />
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Extension.extension">
+      <path value="Extension.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Extension.extension:Gravidity">
+      <path value="Extension.extension" />
+      <sliceName value="Gravidity" />
+    </element>
+    <element id="Extension.extension:Gravidity.url">
+      <path value="Extension.extension.url" />
+      <fixedUri value="Gravidity" />
+    </element>
+    <element id="Extension.extension:Gravidity.value[x]:valueInteger">
+      <path value="Extension.extension.valueInteger" />
+      <sliceName value="valueInteger" />
+      <type>
+        <code value="integer" />

```

Comment:
Shall we provide a bit more context in the extension as well? Like the conceptdescription, alias in the extension as well. Or do you think that is unnecessary/duplicate

Reply:
Deze extensie moet weg, dat is nog een eerdere aanpak waar we uiteindelijk niet voor gekozen hebben.